rts/lra-examples/api-participant-with-coordinator/pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2015 Red Hat, Inc. and/or its affiliates.
```

Comment:
year wrong

Reply:
This was a cut and paste error. We should not have copyrights on quickstarts (the intent of a quickstart is for people to quickly get up and running by cutting and pasting and copyrights will hinder that approach).